lang/cs/Org.Apache.REEF.Network/Group/Operators/IGroupCommOperator.cs

```diff
@@ -39,5 +39,10 @@ public interface IGroupCommOperator<T>
         /// The operator version number.
         /// </summary>
         int Version { get; }
+
+        /// <summary>
+        /// It does necessary checks to make sure the operator is ready to send/receive messages from/to tasks
+        /// </summary>
+        void Initialize();
```

Comment:
This method is to make sure all tasks in the topology are ready. I might rename it like IsReady() or StatusChecking(). It cannot be in constructor because it may cause deadlock.